graphics/managed_surface.h

```diff
@@ -684,7 +684,7 @@ class ManagedSurface {
 	 * Get the palette array.
 	 */
 	WARN_DEPRECATED("Use grabPalette instead")
-	const uint32 *getPalette() const {
+	const byte *getPalette() const {
```

Comment:
wouldn't this break every usage of this method? because the callers expects this to be RGBA - but it's not RGB only.

Reply:
ah, I see https://github.com/scummvm/scummvm/pull/4731 now...

Reply:
@mgerhardy : There are only a very small number of usages of this currently, basically just the AGS engine code which is causing a deprecation warning. I was hoping this API could be removed completely if that usage could be refactored and the need for it removed from the rawBlit API... which PR #4731 appears to accomplish.